rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/config/RoscoHelmConfigurationProperties.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Grab Holdings, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.rosco.manifests.config;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties("helm")
+@Data
+public class RoscoHelmConfigurationProperties {
```

Comment:
Do we expect users to need to override these executable paths?  We didn't provide the option when we only supported helm 2 baking; does adding support for helm 3 change this?  I ask because anything we add to configuration properties becomes a public contract that we need to indefinitely support so want to avoid adding things "just in case" when defaults will work.

Reply:
Yeah, I was quite hesitant on doing this initially, the reason why this might be useful is to support Rosco deployments that does not use the official image. `helm3` is not the official binary name, it is renamed from `helm` so that we can install them side-by-side as they are not 100% backwards compatible.
ref: https://github.com/spinnaker/rosco/commit/5da1ba05b20593a24803e55fcc44fff18ae76136

Reply:
I've run into quite a few situations where, even if users are running the OSS images, they need to override a particular executable to fix a bug more quickly than we can cut a release. In particular, this happens a lot with Packer. I'm a +1 for making this easier on users.

Reply:
Sounds good, fine leaving this in given that there are use cases 👍